decidim-dev/lib/decidim/dev/test/rspec_support/content_processing.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module Decidim
+  class ContentParsers::DummyFooParser < ContentParsers::BaseParser
+    def rewrite
+      content.gsub("foo", "_foo_")
+    end
+
+    def metadata
+      content.scan("foo").size
+    end
+  end
+
+  class ContentRenderers::DummyFooRenderer < ContentRenderers::BaseRenderer
+    def render
+      content.gsub("_foo_", "foo")
```

Comment:
Can you make it render to something else, so the initial content before parsing and the final one are different?

Reply:
Also for the `DummyBarRenderer`